tests/regression_tests/dagmc/legacy/test.py

```diff
@@ -8,42 +8,48 @@
     not openmc.lib._dagmc_enabled(),
     reason="DAGMC CAD geometry is not enabled.")
 
-def test_dagmc():
-    model = openmc.model.Model()
+class DAGMCLegacyTest(PyAPITestHarness):
+
+    def _build_inputs(self):
```

Comment:
You actually don't need to subclass `PyAPITestHarness` just to change the model. You can pass a `Model` object as the second argument when instantiating `PyAPITestHarness(...)`. See the `void` regression test as an example.